packages/next/cms/components/Viz/Table.js

```diff
@@ -1,4 +1,6 @@
+import React from "react";
 
+/** TODO */
```

Comment:
I'm okay not doing this one for this release, but let's make sure we remember that we need it for OEC 🧠

Reply:
didn't know we needed it for the OEC, noted